src/main/kotlin/fi/metatavu/timebank/api/severa/SeveraService.kt

```diff
@@ -0,0 +1,88 @@
+package fi.metatavu.timebank.api.severa
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import fi.metatavu.timebank.api.severa.models.SeveraAccessToken
+import fi.metatavu.timebank.model.User
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import org.eclipse.microprofile.config.inject.ConfigProperty
+import org.slf4j.Logger
+import javax.enterprise.context.RequestScoped
+import javax.inject.Inject
+
+@RequestScoped
+class SeveraService {
+    @ConfigProperty(name = "severa.base.url")
+    lateinit var severaBaseUrl: String
+
+    @ConfigProperty(name = "severa.client.id")
+    lateinit var severaClientId: String
+
+    @ConfigProperty(name = "severa.client.secret")
+    lateinit var severaClientSecret: String
+
+    private var severaAccessToken: SeveraAccessToken? = null
```

Comment:
These should not be needed in this class. The access token should be stored within the `SeveraAccessTokenContainer` as described in that classes review.